MH4U Dex/Images.xcassets/Region Icons/ancestral_steppe-icon.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "scale" : "1x",
+      "filename" : "Image.png"
```

Comment:
Did this image come from somewhere?  Are there any potential licensing issues with including it?